.github/workflows/codeql-java-analysis.yml

```diff
@@ -7,7 +7,7 @@ name: "CodeQL Java"
 
 on:
   push:
-    branches: [main]
+    branches-ignore: [main]
```

Comment:
@andreaTP I'd suggest the same approach as before and schedule the CodeQL analysis on nightly basis for push events.

Same for the other changes above.

Reply:
commented above.